hail/python/hail/table.py

```diff
@@ -554,6 +555,44 @@ def parallelize(cls, rows, schema=None, key=None, n_partitions=None, *,
             table = table.key_by(*key)
         return table
 
+    @staticmethod
+    @typecheck(
+        contexts=expr_stream(expr_any),
```

Comment:
prefer expr_array here. We try not to expose streams much at the Python level -- just wrap the contexts IR in ToStream when creating the TableGen IR

Reply:
How about the body return type?